contributing.md

```diff
@@ -4,7 +4,7 @@ We're really excited that you are interested in contributing to the Envato Marke
 
 ## Issue Reporting Guidelines
 
-- The issue list of this repo is **exclusively** for bug reports and feature requests. For simple questions, please use [Gitter](https://gitter.im/envato/wp-envato-market).
+- The issue list of this repo is **exclusively** for bug reports and feature requests. For simple questions, please use [Gitter](https://gitter.im/envato/envato-market).
```

Comment:
Gitter references should be remove it's not being used.

Reply:
Thanks.